man/redfishpower.8.in

```diff
@@ -27,16 +27,16 @@ Set extra HEADER to use.  Typically is Content-Type:application/json.
 Set Redfish path for obtaining power status.  Typically is redfish/v1/Systems/1.
 .TP
 .I "-O, --onpath string"
-Set Redfish path for performing power on.  Typically is redfish/v1/Systems/1/Actions/ComputerSystem.Reset.
+Set default Redfish path for performing power on.  Typically is redfish/v1/Systems/1/Actions/ComputerSystem.Reset.
 .TP
 .I "-F, --offpath string"
-Set Redfish path for performing power off.  Typically is redfish/v1/Systems/1/Actions/ComputerSystem.Reset.
+Set default Redfish path for performing power off.  Typically is redfish/v1/Systems/1/Actions/ComputerSystem.Reset.
 .TP
 .I "-P, --onpostdata string"
-Set Redfish postdata for performing power on.  Typically is {"ResetType":"On"}
+Set default Redfish postdata for performing power on.  Typically is {"ResetType":"On"}
 .TP
 .I "-G, --offpostdata string"
-Set Redfish postdata for performing power off.  Typically is {"ResetType":"ForceOff"}
+Set default Redfish postdata for performing power off.  Typically is {"ResetType":"ForceOff"}
 .TP
 .I "-v, --verbose"
 Increase output verbosity.  Can be specified multiple times.
```

Comment:
Just curious, why do we have all these options?  It seems unlikely that someone is going to want to repeat these options over and over in a powerman.conf.

Reply:
this is legacy from the original redfishpower implementation.  In order to help with testing I would set all of these paths on the command line.  Perhaps they are becoming less and less useful and should be removed?

Reply:
Just noise in the documentation if nobody is going to use them.